nala/utils/pr.run.sh

```diff
@@ -33,6 +33,7 @@ toRepoName=${repoParts[1]}
 prRepo=${prRepo:-$toRepoName}
 prOrg=${prOrg:-$toRepoOrg}
 
+# TODO ADD HLX5 SUPPORT
```

Comment:
?

Reply:
I'll need @skumar09 help here - to ensure we test on `aem.page/aem.live` for migrated consumers and `hlx.page/hlx.live` for non migrated ones.